monitor/views.py

```diff
@@ -89,11 +98,58 @@ def monitor_home(request):
              "sounds_analysis_failed_count": sounds_analysis_failed_count,
              "sounds_analysis_skipped_count": sounds_analysis_skipped_count,
              "gearman_status": gearman_status,
-             "sounds_in_moderators_queue_count": sounds_in_moderators_queue_count}
+             "sounds_in_moderators_queue_count": sounds_in_moderators_queue_count
+    }
 
     return render(request, 'monitor/monitor.html', tvars)
 
 
+@cache_page(60 * 60 * 24)
+def stats_ajax(request):
+    last_week = datetime.datetime.now()-datetime.timedelta(weeks=169)
+
+    new_sounds_mod = sounds.models.Sound.objects\
+            .filter(created__gt=last_week, moderation_date__isnull=False)\
+            .extra(select={'day': 'date(moderation_date)'}).values('day')\
+            .order_by().annotate(Count('id'))
```

Comment:
The way of doing a group by in django is by using annotate after .values:

http://stackoverflow.com/a/629691/611467